src/analysis/timeline/distributionToCred.js

```diff
@@ -49,15 +56,14 @@ export function distributionToCred(
     }
     cred[i] = new Array(intervals.length);
   }
-  if (scoringNodeIndices.length === 0) {
-    throw new Error("no nodes matched scoringNodePrefix");
-  }
 
   return ds.map(({interval, distribution, intervalWeight}) => {
     const intervalTotalScore = sum(
       scoringNodeIndices.map((x) => distribution[x])
     );
-    const intervalNormalizer = intervalWeight / intervalTotalScore;
+
+    const intervalNormalizer =
+      intervalTotalScore == 0 ? 0 : intervalWeight / intervalTotalScore;
```

Comment:
Now that I'm reading this on review, I see that the code I suggested to you doesn't quite match our best practices. In general we use the type-coercing equality operator (`==`) only when comparing to `null`, since in that case we really do want to take advantage of the fact that `null == undefined`. In many other cases, the type coercion just creates opportunities for new unexpected behavior, e.g. the fact that `[] == 0` is likely to be surprising.

However, I won't block this review on the nit; this is the kind of thing that we should have a [lint rule](https://eslint.org/docs/rules/eqeqeq) for. I'll follow up with a lint rule.

Reply:
cf https://github.com/sourcecred/sourcecred/pull/1579